Can't build uberjar

Dear all,

following #6223, I've checked out the master branch, with the corrections applied (including commit 984728aedb9c5e0a97cc0e764d3bd654cf41ec2a). When trying to build the uberjar, the following error ocurrs. Environment is the same - no changes.

**git log:**

commit 984728aedb9c5e0a97cc0e764d3bd654cf41ec2a
Author: Kyle Doherty <kdoh@users.noreply.github.com>
Date:   Mon Oct 23 18:05:25 2017 -0700

    webpack 3 uglifyjs compatibility (#6242)


git status:

On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean

**bin/build:**

rodrigo@ubuntu:~/Projetos/metabase$ bin/build 
Tagging uberjar with version 'v0.27.0-snapshot 7d178fd master 2017-10-25'...
Running 'yarn' to download javascript dependencies...
yarn install v1.2.0
$ echo $npm_execpath | grep -q yarn || echo '\033[0;33mSorry, npm is not supported. Please use Yarn (https://yarnpkg.com/).\033[0m'
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 2.77s.
Running 'webpack' with NODE_ENV=production assemble and minify frontend assets...
Node#before is deprecated. Use Node#raws.before
Hash: 284a6c7689bace5b42bd
Version: webpack 3.8.1
Time: 80742ms
                                                     Asset       Size  Chunks                    Chunk Names
                      1.bundle.js.map?284a6c7689bace5b42bd     156 kB       1  [emitted]         
                          0.bundle.js?284a6c7689bace5b42bd    31.5 kB       0  [emitted]         
                          2.bundle.js?284a6c7689bace5b42bd     120 kB       2  [emitted]         
                   app-main.bundle.js?284a6c7689bace5b42bd    5.66 MB       3  [emitted]  [big]  app-main
                 app-public.bundle.js?284a6c7689bace5b42bd     2.2 MB       4  [emitted]  [big]  app-public
                  app-embed.bundle.js?284a6c7689bace5b42bd     2.2 MB       5  [emitted]  [big]  app-embed
                     styles.bundle.js?284a6c7689bace5b42bd    2.69 kB       6  [emitted]         styles
                     vendor.bundle.js?284a6c7689bace5b42bd     6.5 MB       7  [emitted]  [big]  vendor
      app-main.bundle.css?2587979a4e968c2a4993ea9f9daf4466     100 kB       3  [emitted]         app-main
    app-public.bundle.css?c113f5bbaafc552df35bc5da39e5c769    80.7 kB       4  [emitted]         app-public
     app-embed.bundle.css?c113f5bbaafc552df35bc5da39e5c769    80.7 kB       5  [emitted]         app-embed
        styles.bundle.css?15fe91d57cb8a506d5712d67f804ea96     119 kB       6  [emitted]         styles
        vendor.bundle.css?83382c49828be91adef81f2bdc6f8b2c      16 kB       7  [emitted]         vendor
                      0.bundle.js.map?284a6c7689bace5b42bd    40.8 kB       0  [emitted]         
                          1.bundle.js?284a6c7689bace5b42bd     106 kB       1  [emitted]         
                      2.bundle.js.map?284a6c7689bace5b42bd     137 kB       2  [emitted]         
               app-main.bundle.js.map?284a6c7689bace5b42bd     5.2 MB       3  [emitted]  [big]  app-main
  app-main.bundle.css.map?2587979a4e968c2a4993ea9f9daf4466  129 bytes       3  [emitted]         app-main
             app-public.bundle.js.map?284a6c7689bace5b42bd    2.25 MB       4  [emitted]  [big]  app-public
app-public.bundle.css.map?c113f5bbaafc552df35bc5da39e5c769  131 bytes       4  [emitted]         app-public
              app-embed.bundle.js.map?284a6c7689bace5b42bd    2.25 MB       5  [emitted]  [big]  app-embed
 app-embed.bundle.css.map?c113f5bbaafc552df35bc5da39e5c769  130 bytes       5  [emitted]         app-embed
                 styles.bundle.js.map?284a6c7689bace5b42bd    3.37 kB       6  [emitted]         styles
    styles.bundle.css.map?15fe91d57cb8a506d5712d67f804ea96  127 bytes       6  [emitted]         styles
                 vendor.bundle.js.map?284a6c7689bace5b42bd    7.87 MB       7  [emitted]  [big]  vendor
    vendor.bundle.css.map?83382c49828be91adef81f2bdc6f8b2c  127 bytes       7  [emitted]         vendor
                                          ../../index.html    4.47 kB          [emitted]         
                                         ../../public.html    4.47 kB          [emitted]         
                                          ../../embed.html    4.47 kB          [emitted]         
  [25] ./lib/dom.js 10.4 kB {3} {4} {5} [built]
 [104] ./redux/app.js 1.11 kB {3} {4} {5} [built]
 [123] ./redux/user.js 3.83 kB {3} {4} {5} [built]
 [200] ./lib/api.js 7.93 kB {3} {4} {5} [built]
 [346] ./app.js 3.14 kB {3} {4} {5} [built]
 [749] ./reducers-public.js 824 bytes {4} {5} [built]
[1149] ./app-main.js 2.57 kB {3} [built]
[1763] ./routes.jsx 30.9 kB {3} [built]
[2174] ./reducers-main.js 3.17 kB {3} [built]
[2181] ./app-public.js 395 bytes {4} [built]
[2182] ./routes-public.jsx 1.73 kB {4} [built]
[2183] ./app-embed.js 672 bytes {5} [built]
[2184] ./routes-embed.jsx 1.74 kB {5} [built]
[2185] ./css/index.css 2.54 kB {6} [built]
[2297] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/index.css 127 kB [built]
    + 2283 hidden modules

WARNING in ./public/components/EmbedFrame.jsx
87:18-25 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./public/components/EmbedFrame.jsx
 @ ./public/containers/PublicQuestion.jsx
 @ ./routes-public.jsx
 @ ./app-public.js
Child html-webpack-plugin for "../../index.html":
     1 asset
       [0] /home/rodrigo/Projetos/metabase/node_modules/html-webpack-plugin/lib/loader.js!/home/rodrigo/Projetos/metabase/resources/frontend_client/index_template.html 4.39 kB {0} [built]
       [2] (webpack)/buildin/global.js 488 bytes {0} [built]
       [3] (webpack)/buildin/module.js 495 bytes {0} [built]
        + 1 hidden module
Child html-webpack-plugin for "../../public.html":
     1 asset
       [0] /home/rodrigo/Projetos/metabase/node_modules/html-webpack-plugin/lib/loader.js!/home/rodrigo/Projetos/metabase/resources/frontend_client/index_template.html 4.39 kB {0} [built]
       [2] (webpack)/buildin/global.js 488 bytes {0} [built]
       [3] (webpack)/buildin/module.js 495 bytes {0} [built]
        + 1 hidden module
Child html-webpack-plugin for "../../embed.html":
     1 asset
       [0] /home/rodrigo/Projetos/metabase/node_modules/html-webpack-plugin/lib/loader.js!/home/rodrigo/Projetos/metabase/resources/frontend_client/index_template.html 4.39 kB {0} [built]
       [2] (webpack)/buildin/global.js 488 bytes {0} [built]
       [3] (webpack)/buildin/module.js 495 bytes {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!../../../node_modules/react-virtualized/styles.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!../../../node_modules/leaflet/dist/leaflet.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Popover.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Calendar.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/ColumnarSelector.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/LoadingSpinner.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Toggle.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!containers/SaveQuestionModal.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./containers/SaveQuestionModal.css 689 bytes {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/SortableItemList.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!tutorial/PageFlag.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./tutorial/PageFlag.css 5.02 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/Legend.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/FunnelNormal.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/LineAreaBarChart.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/ChartWithLegend.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/visualizations/PieChart.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./visualizations/visualizations/PieChart.css 1.25 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/Table.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/TableInteractive.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/visualizations/Scalar.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./visualizations/visualizations/Scalar.css 1.06 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!public/components/EmbedFrame.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!query_builder/components/NativeQueryEditor.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!query_builder/components/expressions/ExpressionEditorTextfield.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!query_builder/components/expressions/TokenizedExpression.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!admin/permissions/components/FixedHeaderGrid.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Breadcrumbs.css:
       [1] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/Reference.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./reference/Reference.css 18.3 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/List.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/QueryButton.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Sidebar.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/LabelIcon.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!containers/UndoListing.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./containers/UndoListing.css 5.34 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!parameters/components/ParameterWidget.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!dashboard/components/RefreshWidget.css:
       [1] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!dashboard/containers/DashCardCardParameterMapper.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./dashboard/containers/DashCardCardParameterMapper.css 2.25 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/UsefulQuestions.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/Detail.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/ReferenceHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/EditButton.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/EditHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/RevisionMessageModal.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/Field.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/Formula.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/FieldToGroupBy.css:
       [1] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/GuideDetailEditor.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/GuideEditSection.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/Labels.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/containers/LabelEditorForm.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/containers/LabelEditorForm.css 2.12 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/SearchHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/List.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/Questions.css 912 bytes {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/ActionHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [3] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/Questions.css 912 bytes {0} [built]
        + 3 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/LabelPicker.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/containers/EditLabels.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/Questions.css 912 bytes {0} [built]
       [3] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/containers/EditLabels.css 5.25 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/LabelIconPicker.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!css/index.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/index.css 127 kB {0} [built]
        + 1 hidden module
Sample Dataset already generated.
Running 'lein uberjar'...
Compiling metabase.driver.bigquery
log4j:ERROR Could not instantiate class [metabase.logger.Appender].
java.lang.ClassNotFoundException: metabase.logger.Appender
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.apache.log4j.helpers.Loader.loadClass(Loader.java:198)
	at org.apache.log4j.helpers.OptionConverter.instantiateByClassName(OptionConverter.java:327)
	at org.apache.log4j.helpers.OptionConverter.instantiateByKey(OptionConverter.java:124)
	at org.apache.log4j.PropertyConfigurator.parseAppender(PropertyConfigurator.java:785)
	at org.apache.log4j.PropertyConfigurator.parseCategory(PropertyConfigurator.java:768)
	at org.apache.log4j.PropertyConfigurator.configureRootCategory(PropertyConfigurator.java:648)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:514)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:580)
	at org.apache.log4j.helpers.OptionConverter.selectAndConfigure(OptionConverter.java:526)
	at org.apache.log4j.LogManager.<clinit>(LogManager.java:127)
	at org.slf4j.impl.Log4jLoggerFactory.<init>(Log4jLoggerFactory.java:66)
	at org.slf4j.impl.StaticLoggerBinder.<init>(StaticLoggerBinder.java:72)
	at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:45)
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:412)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:357)
	at clojure.tools.logging$eval219$reify__220.get_logger(logging.clj:288)
	at metabase.util$fn__3460.invokeStatic(util.clj:33)
	at metabase.util$fn__3460.invoke(util.clj:33)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:551)
	at metabase.db$loading__5569__auto____2302.invoke(db.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:512)
	at metabase.models.database$loading__5569__auto____1993.invoke(database.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:703)
	at metabase.driver$loading__5569__auto____1124.invoke(driver.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:930)
	at metabase.driver.bigquery$loading__5569__auto____36.invoke(bigquery.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$compile$fn__5682.invoke(core.clj:5903)
	at clojure.core$compile.invokeStatic(core.clj:5903)
	at clojure.core$compile.invoke(core.clj:5895)
	at user$eval20$fn__29.invoke(form-init2085043060456206224.clj:1)
	at user$eval20.invokeStatic(form-init2085043060456206224.clj:1)
	at user$eval20.invoke(form-init2085043060456206224.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
log4j:ERROR Could not instantiate appender named "metabase".
10-26 06:32:28 INFO metabase.util :: Loading Metabase...
10-26 06:32:37 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓 
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
java.lang.ClassNotFoundException: metabase.query_processor.interface.Field, compiling:(generic_sql.clj:1:1)
Exception in thread "main" java.lang.ClassNotFoundException: metabase.query_processor.interface.Field, compiling:(generic_sql.clj:1:1)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3657)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:930)
	at metabase.driver.bigquery$loading__5569__auto____36.invoke(bigquery.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$compile$fn__5682.invoke(core.clj:5903)
	at clojure.core$compile.invokeStatic(core.clj:5903)
	at clojure.core$compile.invoke(core.clj:5895)
	at user$eval20$fn__29.invoke(form-init2085043060456206224.clj:1)
	at user$eval20.invokeStatic(form-init2085043060456206224.clj:1)
	at user$eval20.invoke(form-init2085043060456206224.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
Caused by: java.lang.ClassNotFoundException: metabase.query_processor.interface.Field
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at clojure.lang.RT.classForName(RT.java:2168)
	at clojure.lang.RT.classForNameNonLoading(RT.java:2181)
	at metabase.driver.generic_sql$loading__5569__auto____23054.invoke(generic_sql.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	... 63 more
Compilation failed: Subprocess failed

Dear khod,

I've upgraded my environment as follows but still face the same issue (using latest master branch):

rodrigo@ubuntu:~/Projetos/metabase$ java -version
java version "1.8.0_151"
Java(TM) SE Runtime Environment (build 1.8.0_151-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.151-b12, mixed mode)
rodrigo@ubuntu:~/Projetos/metabase$ node -v
v8.8.1
rodrigo@ubuntu:~/Projetos/metabase$ yarn -v
1.2.1





rodrigo@ubuntu:~/Projetos/metabase$ bin/build
Tagging uberjar with version 'v0.27.0-snapshot a228f73 master 2017-10-26'...
Running 'yarn' to download javascript dependencies...
yarn install v1.2.1
$ echo $npm_execpath | grep -q yarn || echo '\033[0;33mSorry, npm is not supported. Please use Yarn (https://yarnpkg.com/).\033[0m'
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 1.42s.
Running 'webpack' with NODE_ENV=production assemble and minify frontend assets...
Node#before is deprecated. Use Node#raws.before
Hash: 7478c5f39301d13cb3e5
Version: webpack 3.8.1
Time: 78886ms
                                                     Asset       Size  Chunks                    Chunk Names
                      1.bundle.js.map?7478c5f39301d13cb3e5     156 kB       1  [emitted]         
                          0.bundle.js?7478c5f39301d13cb3e5    31.5 kB       0  [emitted]         
                          2.bundle.js?7478c5f39301d13cb3e5     120 kB       2  [emitted]         
                   app-main.bundle.js?7478c5f39301d13cb3e5    5.66 MB       3  [emitted]  [big]  app-main
                 app-public.bundle.js?7478c5f39301d13cb3e5     2.2 MB       4  [emitted]  [big]  app-public
                  app-embed.bundle.js?7478c5f39301d13cb3e5     2.2 MB       5  [emitted]  [big]  app-embed
                     styles.bundle.js?7478c5f39301d13cb3e5    2.69 kB       6  [emitted]         styles
                     vendor.bundle.js?7478c5f39301d13cb3e5     6.5 MB       7  [emitted]  [big]  vendor
      app-main.bundle.css?2587979a4e968c2a4993ea9f9daf4466     100 kB       3  [emitted]         app-main
    app-public.bundle.css?c113f5bbaafc552df35bc5da39e5c769    80.7 kB       4  [emitted]         app-public
     app-embed.bundle.css?c113f5bbaafc552df35bc5da39e5c769    80.7 kB       5  [emitted]         app-embed
        styles.bundle.css?15fe91d57cb8a506d5712d67f804ea96     119 kB       6  [emitted]         styles
        vendor.bundle.css?83382c49828be91adef81f2bdc6f8b2c      16 kB       7  [emitted]         vendor
                      0.bundle.js.map?7478c5f39301d13cb3e5    40.8 kB       0  [emitted]         
                          1.bundle.js?7478c5f39301d13cb3e5     106 kB       1  [emitted]         
                      2.bundle.js.map?7478c5f39301d13cb3e5     137 kB       2  [emitted]         
               app-main.bundle.js.map?7478c5f39301d13cb3e5     5.2 MB       3  [emitted]  [big]  app-main
  app-main.bundle.css.map?2587979a4e968c2a4993ea9f9daf4466  129 bytes       3  [emitted]         app-main
             app-public.bundle.js.map?7478c5f39301d13cb3e5    2.25 MB       4  [emitted]  [big]  app-public
app-public.bundle.css.map?c113f5bbaafc552df35bc5da39e5c769  131 bytes       4  [emitted]         app-public
              app-embed.bundle.js.map?7478c5f39301d13cb3e5    2.25 MB       5  [emitted]  [big]  app-embed
 app-embed.bundle.css.map?c113f5bbaafc552df35bc5da39e5c769  130 bytes       5  [emitted]         app-embed
                 styles.bundle.js.map?7478c5f39301d13cb3e5    3.37 kB       6  [emitted]         styles
    styles.bundle.css.map?15fe91d57cb8a506d5712d67f804ea96  127 bytes       6  [emitted]         styles
                 vendor.bundle.js.map?7478c5f39301d13cb3e5    7.87 MB       7  [emitted]  [big]  vendor
    vendor.bundle.css.map?83382c49828be91adef81f2bdc6f8b2c  127 bytes       7  [emitted]         vendor
                                          ../../index.html    4.47 kB          [emitted]         
                                         ../../public.html    4.47 kB          [emitted]         
                                          ../../embed.html    4.47 kB          [emitted]         
  [25] ./lib/dom.js 10.4 kB {3} {4} {5} [built]
 [104] ./redux/app.js 1.11 kB {3} {4} {5} [built]
 [123] ./redux/user.js 3.83 kB {3} {4} {5} [built]
 [200] ./lib/api.js 7.93 kB {3} {4} {5} [built]
 [346] ./app.js 3.14 kB {3} {4} {5} [built]
 [749] ./reducers-public.js 824 bytes {4} {5} [built]
[1149] ./app-main.js 2.57 kB {3} [built]
[1763] ./routes.jsx 30.9 kB {3} [built]
[2174] ./reducers-main.js 3.17 kB {3} [built]
[2181] ./app-public.js 395 bytes {4} [built]
[2182] ./routes-public.jsx 1.73 kB {4} [built]
[2183] ./app-embed.js 672 bytes {5} [built]
[2184] ./routes-embed.jsx 1.74 kB {5} [built]
[2185] ./css/index.css 2.54 kB {6} [built]
[2297] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/index.css 127 kB [built]
    + 2283 hidden modules

WARNING in ./public/components/EmbedFrame.jsx
87:18-25 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./public/components/EmbedFrame.jsx
 @ ./public/containers/PublicDashboard.jsx
 @ ./routes-embed.jsx
 @ ./app-embed.js
Child html-webpack-plugin for "../../index.html":
     1 asset
       [0] /home/rodrigo/Projetos/metabase/node_modules/html-webpack-plugin/lib/loader.js!/home/rodrigo/Projetos/metabase/resources/frontend_client/index_template.html 4.39 kB {0} [built]
       [2] (webpack)/buildin/global.js 488 bytes {0} [built]
       [3] (webpack)/buildin/module.js 495 bytes {0} [built]
        + 1 hidden module
Child html-webpack-plugin for "../../public.html":
     1 asset
       [0] /home/rodrigo/Projetos/metabase/node_modules/html-webpack-plugin/lib/loader.js!/home/rodrigo/Projetos/metabase/resources/frontend_client/index_template.html 4.39 kB {0} [built]
       [2] (webpack)/buildin/global.js 488 bytes {0} [built]
       [3] (webpack)/buildin/module.js 495 bytes {0} [built]
        + 1 hidden module
Child html-webpack-plugin for "../../embed.html":
     1 asset
       [0] /home/rodrigo/Projetos/metabase/node_modules/html-webpack-plugin/lib/loader.js!/home/rodrigo/Projetos/metabase/resources/frontend_client/index_template.html 4.39 kB {0} [built]
       [2] (webpack)/buildin/global.js 488 bytes {0} [built]
       [3] (webpack)/buildin/module.js 495 bytes {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!../../../node_modules/react-virtualized/styles.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!../../../node_modules/leaflet/dist/leaflet.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Popover.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Calendar.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/ColumnarSelector.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/LoadingSpinner.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Toggle.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!containers/SaveQuestionModal.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./containers/SaveQuestionModal.css 689 bytes {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/SortableItemList.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!tutorial/PageFlag.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./tutorial/PageFlag.css 5.02 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/Legend.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/FunnelNormal.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/LineAreaBarChart.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/ChartWithLegend.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/visualizations/PieChart.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./visualizations/visualizations/PieChart.css 1.25 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/Table.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/components/TableInteractive.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!visualizations/visualizations/Scalar.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./visualizations/visualizations/Scalar.css 1.06 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!public/components/EmbedFrame.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!query_builder/components/NativeQueryEditor.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!query_builder/components/expressions/ExpressionEditorTextfield.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!query_builder/components/expressions/TokenizedExpression.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!admin/permissions/components/FixedHeaderGrid.css:
       2 modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Breadcrumbs.css:
       [1] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/Reference.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./reference/Reference.css 18.3 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/List.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/QueryButton.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/Sidebar.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/LabelIcon.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!containers/UndoListing.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./containers/UndoListing.css 5.34 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!parameters/components/ParameterWidget.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!dashboard/components/RefreshWidget.css:
       [1] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!dashboard/containers/DashCardCardParameterMapper.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./dashboard/containers/DashCardCardParameterMapper.css 2.25 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/UsefulQuestions.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/Detail.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/ReferenceHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/EditButton.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/EditHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/RevisionMessageModal.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/Field.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/Formula.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/FieldToGroupBy.css:
       [1] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/GuideDetailEditor.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!reference/components/GuideEditSection.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/Labels.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/containers/LabelEditorForm.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/containers/LabelEditorForm.css 2.12 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!components/SearchHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/List.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/Questions.css 912 bytes {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/ActionHeader.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [3] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/Questions.css 912 bytes {0} [built]
        + 3 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/LabelPicker.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/containers/EditLabels.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
       [2] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/Questions.css 912 bytes {0} [built]
       [3] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./questions/containers/EditLabels.css 5.25 kB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!questions/components/LabelIconPicker.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/core/index.css 56.1 kB {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin ../../../node_modules/extract-text-webpack-plugin/dist ../../../node_modules/css-loader/index.js??ref--3-2!../../../node_modules/postcss-loader/lib/index.js!css/index.css:
       [0] /home/rodrigo/Projetos/metabase/node_modules/css-loader?{"localIdentName":"[hash:base64:5]","url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/node_modules/postcss-loader/lib!./css/index.css 127 kB {0} [built]
        + 1 hidden module
Sample Dataset already generated.
Running 'lein uberjar'...
Compiling metabase.driver.bigquery
log4j:ERROR Could not instantiate class [metabase.logger.Appender].
java.lang.ClassNotFoundException: metabase.logger.Appender
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.apache.log4j.helpers.Loader.loadClass(Loader.java:198)
	at org.apache.log4j.helpers.OptionConverter.instantiateByClassName(OptionConverter.java:327)
	at org.apache.log4j.helpers.OptionConverter.instantiateByKey(OptionConverter.java:124)
	at org.apache.log4j.PropertyConfigurator.parseAppender(PropertyConfigurator.java:785)
	at org.apache.log4j.PropertyConfigurator.parseCategory(PropertyConfigurator.java:768)
	at org.apache.log4j.PropertyConfigurator.configureRootCategory(PropertyConfigurator.java:648)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:514)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:580)
	at org.apache.log4j.helpers.OptionConverter.selectAndConfigure(OptionConverter.java:526)
	at org.apache.log4j.LogManager.<clinit>(LogManager.java:127)
	at org.slf4j.impl.Log4jLoggerFactory.<init>(Log4jLoggerFactory.java:66)
	at org.slf4j.impl.StaticLoggerBinder.<init>(StaticLoggerBinder.java:72)
	at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:45)
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:412)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:357)
	at clojure.tools.logging$eval219$reify__220.get_logger(logging.clj:288)
	at metabase.util$fn__3460.invokeStatic(util.clj:33)
	at metabase.util$fn__3460.invoke(util.clj:33)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:551)
	at metabase.db$loading__5569__auto____2302.invoke(db.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:512)
	at metabase.models.database$loading__5569__auto____1993.invoke(database.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:703)
	at metabase.driver$loading__5569__auto____1124.invoke(driver.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:930)
	at metabase.driver.bigquery$loading__5569__auto____36.invoke(bigquery.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$compile$fn__5682.invoke(core.clj:5903)
	at clojure.core$compile.invokeStatic(core.clj:5903)
	at clojure.core$compile.invoke(core.clj:5895)
	at user$eval20$fn__29.invoke(form-init4920649744862644229.clj:1)
	at user$eval20.invokeStatic(form-init4920649744862644229.clj:1)
	at user$eval20.invoke(form-init4920649744862644229.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
log4j:ERROR Could not instantiate appender named "metabase".
10-27 06:42:17 INFO metabase.util :: Loading Metabase...
10-27 06:42:27 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓 
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
java.lang.ClassNotFoundException: metabase.query_processor.interface.Field, compiling:(generic_sql.clj:1:1)
Exception in thread "main" java.lang.ClassNotFoundException: metabase.query_processor.interface.Field, compiling:(generic_sql.clj:1:1)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3657)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:930)
	at metabase.driver.bigquery$loading__5569__auto____36.invoke(bigquery.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$compile$fn__5682.invoke(core.clj:5903)
	at clojure.core$compile.invokeStatic(core.clj:5903)
	at clojure.core$compile.invoke(core.clj:5895)
	at user$eval20$fn__29.invoke(form-init4920649744862644229.clj:1)
	at user$eval20.invokeStatic(form-init4920649744862644229.clj:1)
	at user$eval20.invoke(form-init4920649744862644229.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
Caused by: java.lang.ClassNotFoundException: metabase.query_processor.interface.Field
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at clojure.lang.RT.classForName(RT.java:2168)
	at clojure.lang.RT.classForNameNonLoading(RT.java:2181)
	at metabase.driver.generic_sql$loading__5569__auto____23054.invoke(generic_sql.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	... 63 more
Compilation failed: Subprocess failed

--- a/src/metabase/driver/generic_sql.clj
+++ b/src/metabase/driver/generic_sql.clj
@@ -15,6 +15,7 @@
             [metabase.models
              [field :as field]
              [table :as table]]
+            metabase.query-processor.interface
             [metabase.util
              [honeysql-extensions :as hx]
              [ssh :as ssh]])

--- a/src/metabase/driver/generic_sql/query_processor.clj
+++ b/src/metabase/driver/generic_sql/query_processor.clj
@@ -19,7 +19,8 @@
   (:import clojure.lang.Keyword
            [java.sql PreparedStatement ResultSet ResultSetMetaData SQLException]
            [java.util Calendar TimeZone]
-           [metabase.query_processor.interface AgFieldRef BinnedField DateTimeField DateTimeValue Expression ExpressionRef Field FieldLiteral RelativeDateTimeValue Value]))
+           [metabase.query_processor.interface AgFieldRef BinnedField DateTimeField DateTimeValue Expression
+            ExpressionRef Field FieldLiteral RelativeDateTimeValue Value]))
 
 (def ^:dynamic *query*
   "The outer query currently being processed."

--- a/src/metabase/models/params.clj
+++ b/src/metabase/models/params.clj
@@ -1,6 +1,7 @@
 (ns metabase.models.params
   "Utility functions for dealing with parameters for Dashboards and Cards."
   (:require [metabase.query-processor.middleware.expand :as ql]
+            metabase.query-processor.interface
             [metabase.util :as u]
             [toucan.db :as db])
   (:import metabase.query_processor.interface.FieldPlaceholder))

--- a/test_resources/log4j.properties
+++ b/test_resources/log4j.properties
@@ -1,4 +1,4 @@
-log4j.rootLogger=WARN, console
+log4j.rootLogger=ERROR, console
 
 # log to the console
 log4j.appender.console=org.apache.log4j.ConsoleAppender
@@ -15,12 +15,7 @@ log4j.appender.file.layout=org.apache.log4j.PatternLayout
 log4j.appender.file.layout.ConversionPattern=%d [%t] %-5p%c - %m%n
 
 # customizations to logging by package
-log4j.logger.com.mchange=ERROR
-log4j.logger.org.eclipse.jetty.server.HttpChannel=ERROR
 log4j.logger.metabase=ERROR
 log4j.logger.metabase.test-setup=INFO
-log4j.logger.metabase.sync=DEBUG
-log4j.logger.metabase.task.sync-databases=INFO
 log4j.logger.metabase.test.data.datasets=INFO
-log4j.logger.metabase.test.data=DEBUG
-log4j.logger.metabase.util.encryption=INFO
+log4j.logger.metabase.test.data=INFO

